geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/geotools/Conversions.scala

```diff
@@ -246,5 +252,27 @@ object RichSimpleFeatureType {
       Seq(GEOMESA_PREFIX) ++ userData[String](USER_DATA_PREFIX).map(_.split(",")).getOrElse(Array.empty)
 
     def userData[T](key: AnyRef): Option[T] = Option(sft.getUserData.get(key).asInstanceOf[T])
+
+    def getKeywords: java.util.Set[String] = {
```

Comment:
ah, possibly... although aren't we joining it with a scala Set in the data store trait?